### ISSUE TITLE
macro error

### ISSUE THREAD

**Issue by TLINDEN** [karthink/calc-transient#1]
Hi,

I wanted to try this out as it sounds something good to have, but I get:
```
Eager macro-expansion failure: (error "Mandatory ARGLIST is missing") [2 times]
transient--expand-define-args: Mandatory ARGLIST is missing
```
when I try to load the el file.

**Comment by karthink** [comment]
Hi @TLINDEN, it probably needs to be updated for the latest version of transient.

I'm not actively working on this interface any more, as just opening up a python or Julia shell works better for me (scales better too) these days. If you want to take a crack at improving calc's interface, please go ahead. PRs are welcome.

**Comment by fleimgruber** [comment]
@TLINDEN Did you try again more recently? I was able to load the package without error. Maybe it was an unfortunate timing during a stabilization phase of transient upstream?

**Comment by TLINDEN** [comment]
Ahm, no I didn't, but I did now. It loads w/o any errors, but I can't use it :)

As I understand, I have to start `calc` and then execute `calc-transient-mode`, do I? Anyway, if I do, it loads, but I can't use it. If I enter `?` I get:

`transient-setup: Suffix command calc-execute-extended-command is not defined or autoloaded`

**Comment by fleimgruber** [comment]
Yes, `calc` and then `calc-transient-mode` is what I do as well.

Which version of transient do you use? I am using 0.8.6 from https://github.com/magit/transient/commit/52f737a696cad846d528a5a06e412f740a1de38a - it should also be available from MELPA.
